config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
+  namespace :api do
+    namespace :v1, defaults: { format: :json } do
+      resources :greetings, only: [:index]
+    end
+  end
```

Comment:
- [ ] I create one greeting and test in if ı can see the greeting in the server or not but it is giving an error. Kindly fix it.
![image](https://user-images.githubusercontent.com/38635158/180606668-616003fb-1d91-49ec-8529-e18033a1f5ad.png)